brain/task_executor.py

```
@@ -419,6 +426,246 @@ def _extract_latest_user_intent(self, conversation: str) -> str:
+        return latest[:300]
+
+    @staticmethod
+    def _sanitize_correction_text(text: str) -> str:
+        cleaned = str(text or "")
+        cleaned = cleaned.replace("\r", " ").replace("\n", " ")
+        patterns = [
+            (r"(?i)(password|passwd|pwd)\s*[:=]\s*\S+", r"\1=[REDACTED_PASSWORD]"),
+            (r"(?i)(token|api[_-]?key|access[_-]?token|refresh[_-]?token)\s*[:=]\s*\S+", r"\1=[REDACTED_TOKEN]"),
+            (r"(?i)(cookie)\s*[:=]\s*\S+", r"\1=[REDACTED_COOKIE]"),
+            (
+                r"(?i)(\b(?:otp|pin|verification(?:\s+code)?|sms\s*code|one[-\s]?time(?:\s+password|\s+code)?|验证码|校验码|短信码|动态码)\b(?:\s*(?:is|为|是))?[\s:：=#-]{0,6})\d{4,8}\b",
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)